tests/test-metaboxes.php

```diff
@@ -29,8 +29,6 @@ public function test_update_font_stacks() {
 		$this->assertArrayHasKey( 'part', $wp_meta_boxes );
 		$this->assertArrayHasKey( 'metadata', $c->metadata );
 		$this->assertArrayHasKey( 'general-book-information', $c->metadata['metadata'] );
-		$this->assertArrayHasKey( 'additional-catalogue-information', $c->metadata['metadata'] );
```

Comment:
Instead of removing this test, can we add another test that checks if its hidden or not? (Do `update_option( 'pressbooks_show_expanded_metadata' 1);` and test if `additional-catalogue-information` is there.)